### PR TITLE
feat: Update release-kdocs.yml

### PR DESCRIPTION
**Pull request by carlossteinzup**
### Description and Example

Some variable names were changed on AWS and a update was needed on this action

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.
- [x] I have made the documentation changes or I created an issue explaining how to document this change on [Docs issues](https://github.com/ZupIT/beagle-docs/issues). Please link the issue here:

[PR Guide]: https://github.com/ZupIT/beagle/blob/main/doc/contributing/pull_requests.md

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">The title of PR not follow the rules of: https://www.conventionalcommits.org/en/v1.0.0/</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: .github/workflows/release-kdocs.yml [hunk 1]
```diff
@@ -60,4 +60,4 @@ jobs:
 
       - name: S3 BETA upload
         if: ${{ github.ref == 'refs/heads/main' }}
-        run: aws s3 sync --acl public-read --follow-symlinks fastlane/kdocs/public s3://${{ env.BEAGLE_DOCS_AWS_AWS_DOCS_BETA_BUCKET }}/ --delete
\ No newline at end of file
+        run: aws s3 sync --acl public-read --follow-symlinks fastlane/kdocs/public s3://${{ env.BEAGLE_DOCS_AWS_AWS_DOCS_BETA_BUCKET }}/ --delete
```
Comment (hernandazevedozup) [inline .github/workflows/release-kdocs.yml:63]:
```suggestion
        run: aws s3 sync --acl public-read --follow-symlinks fastlane/kdocs/public s3://${{ env.BEAGLE_DOCS_AWS_AWS_DOCS_REFERENCE_BETA_BUCKET }}/ --delete
```

Comment (hernandazevedozup) [reply .github/workflows/release-kdocs.yml:63]: Chante the step "S3 upload" to the variable BEAGLE_DOCS_AWS_AWS_DOCS_REFERENCE_S3_BUCKET

**Review by hernandazevedozup — CHANGES_REQUESTED** [review]
Check the comments

**Review by hernandazevedozup — APPROVED** [review]

**Review by uziasferreirazup — APPROVED** [review]

**Pull request merged**